fenix/app/src/main/java/org/mozilla/fenix/shopping/middleware/ReviewQualityCheckService.kt

```diff
@@ -195,6 +202,23 @@ class DefaultReviewQualityCheckService(
             }
         }
     }
+
+    override suspend fun reportBackInStock(): ReportBackInStockStatusDto? = withContext(Dispatchers.Main) {
+        suspendCoroutine { continuation ->
+            browserStore.state.selectedTab?.let { tab ->
+                tab.engineState.engineSession?.reportBackInStock(
+                    url = tab.content.url,
+                    onResult = {
+                        continuation.resume(it.asEnumOrDefault(ReportBackInStockStatusDto.NOT_DELETED))
```

Comment:
Let's not set a default value in `asEnumOrDefault`

Reply:
Removed default value `it.asEnumOrDefault<ReportBackInStockStatusDto>()`